src/views/collection-view/server.ts

```diff
@@ -94,6 +94,10 @@ export async function getCollectionPageProps(
 
   return {
     props: stripUndefinedProperties({
+      metadata: {
+        title: collection.shortName,
+        description: collection.description,
```

Comment:
🙋🏻 Are these properties on `collection` always defined or are we leveraging `stripUndefinedProperties` to set values to `null`? Same question for `src/views/tutorial-view/server.ts`.

Reply:
I think they should always be defined, but yes we would rely on `stripUndefinedProperties` if they happen to be undefined. 👍